dashboard/app/models/ability.rb

```diff
@@ -221,7 +222,7 @@ def initialize(user)
       ]
 
       # Only custom levels are editable.
-      cannot [:update, :destroy], Level do |level|
+      cannot [:clone, :update, :destroy], Level do |level|
```

Comment:
This refers to the following `custom?` method on `Level`, which does include DSL-defined levels. It could be read as the opposite of "hard-coded."  Based on the comment, it seems levels should be given a user id when they are created on levelbuilder (although that user id may not meaningfully refer to a user in other environments).

https://github.com/code-dot-org/code-dot-org/blob/f0e8fdb5642039af4b75f068bc856185dcfd7cd1/dashboard/app/models/levels/level.rb#L185-L189